setup.py

```diff
@@ -2,6 +2,7 @@
 # where package_data is expected to be str and not unicode.
 from __future__ import absolute_import, division, print_function
 
+import os.path
```

Comment:
I had no idea this was [as controversial as it is](http://stackoverflow.com/questions/2724348/). I'm fine with changing it, but I cringe thinking about all the times in the past that I've done `import os.path`. 😖